draft-ietf-masque-connect-ip.md

```diff
@@ -430,6 +430,26 @@ decremented right before an IP packet is transmitted in an HTTP Datagram. This
 prevents infinite loops in the presence of routing loops, and matches the
 choices in IPsec {{?IPSEC=RFC4301}}.
 
+IPv6 requires that every link have an MTU of at least 1280 bytes
+{{!IPv6=RFC8200}}. Since CONNECT-IP conveys IP packets in HTTP Datagrams and
+those can in turn be sent in QUIC DATAGRAM frames which cannot be fragmented
+{{!DGRAM=RFC8221}}, the MTU of a CONNECT-IP link can be limited by the MTU of
+the QUIC connection that CONNECT-IP is operating over. This can lead to
+situations where the IPv6 minimum link MTU is violated. CONNECT-IP endpoints
+that support IPv6 MUST ensure that the link MTU is at least 1280. This can be
+accomplished using various techniques:
+
+* if HTTP intermediaries are not in use, CONNECT-IP endpoints can pad QUIC
+  INITIAL packets.
```

Comment:
Also here it is unclear if padding the QUIC initial packet is referring to the HTTP/3 connection that MASQUE requests are sent in that must take its own overhead into account and verify that it has room for those 1280 bytes of payload?